mock_drs/database/register_mongodb.py

```diff
@@ -2,78 +2,67 @@
 
 import logging
 from typing import Dict
+import json
 
 from flask import Flask
 from flask_pymongo import ASCENDING, PyMongo
 
-#from app.config.config_parser import get_conf
-#from app.ga4gh.drs.endpoints.get_service_info import get_service_info
+import pymongo
 
+from config.config_parser import get_conf
 
-# Get logger instance
-logger = logging.getLogger(__name__)
+from random import choice
```

Comment:
Put _all_ imports in the import blocks at the top. There has to be a very good reason why imports should not be at the top.

Reply:
will fix this.